blocks/footer/footer.css

```diff
@@ -1,26 +1,178 @@
-footer {
+footer .footer {
   background-color: var(--background-color-footer);
+  max-width: var(--desktop-max-width);
+  padding: var(--gutter-space-section-mobile)  var(--gutter-space-mobile) var(--gutter-space-mobile);
 }
 
-footer .footer {
-  padding: 0 var(--gutter-space-mobile);
-  max-width: var(--desktop-max-width);
-  margin: auto;
+footer .footer ul{
+  padding: 0;
+  list-style: none;
+  margin: 0;
+}
+
+footer .footer .footer-nav .nav-lists {
   display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  list-style: none;
+}
+
+footer .footer .footer-nav .nav-lists > li {
+  position: relative;
+  flex: 1;
+  border-bottom: 1px solid var(--border-color);
+  display: inline-block;
+  padding: 1.063rem 0;
+  color: var(--body-text-color);
+  font-size: var( --body-font-size-xs);
+  font-weight: bold;
+}
+
+footer .footer .footer-nav .nav-lists.nav-lists-with-child > li:first-child {
+  border-top: 1px solid rgba(255, 255, 255, 0.5);
+}
+
+footer .footer .footer-nav .nav-lists > li a {
+  text-decoration: none;
```

Comment:
no need to set text-decoration here, its coming from base theme.

Reply:
done